### DISCUSSION TITLE
Allegro wprowadza w błąd tytułami w ofertach

### DISCUSSION THREAD

**Discussion by GHalbt** [allegro/allegro-api#6578] [Suggestions - rejected]
Wystawiając przez API dziesiątki tysięcy produktów nie ma możliwości sprawdzenia każdej aukcji. Są sytuację gdzie tytuł z bazy danych sklepu lub pliku XML mówi o jednej sztuce produktu a tytuł produktu z katalogu allegro zawiera dopisek X sztuk (to tylko pierwszy przyklad z brzegu, są gorsze patologię). Powoduje to straty u sprzedających, negatywne komentarze za anulowane zamówienia oraz straty allegro w ramach prowizji.

Natychmiast trzeba zrobić opcje aby tytuł produktu z katalogu NIE wyświetlał się jako główny na 3/4 szerokości ekranu a na pewno nie był pierwszy z góry. To nim głównie sugerują się kupujący. Jest to absurdalne ze aukcja końcowo zawiera 2 różne tytuły. Główny, na samej górze jest z katalogu a ten po prawej stronie (w wersji mobilnej ten niżej) to tytuł własny.

![212546253-1fbf37c4-997f-405a-98f8-fbc2e2829131](https://user-images.githubusercontent.com/120689175/212552319-05d276c8-1ffb-4dd0-9103-bb6d776c805c.jpg)

**Comment by MartaNowaczyk** [comment]
Sugestia w API odrzucona. Wyświetlanie tytułu produktu na stronie oferty możesz zgłosić jeszcze przez [formularz kontaktowy](https://allegro.pl/pomoc/kontakt).
